pycytominer/tests/test_cyto_utils/test_load.py

```diff
@@ -196,3 +197,22 @@ def test_load_npz():
         load_npz_locations(
             example_npz_file_locations, location_x_col_index=0, location_y_col_index=2
         )
+
+def test_infer_plate_files():
```

Comment:
```suggestion
def test_is_path_a_parquet_file():
```

To align the name with the function you're testing

Reply:
It also looks like you need to add the test parquet file https://github.com/cytomining/pycytominer/actions/runs/4470828258/jobs/7854910373?pr=262#step:5:91